client/reconfiguration/test/poll_based_state_client_test.cpp

```diff
@@ -160,7 +160,8 @@ class ClientApiTestFixture : public ::testing::Test {
       std::vector<uint8_t> msg_buf;
       concord::messages::serialize(msg_buf, rres);
       auto rep = createReply(msg, msg_buf);
-      client_receiver->onNewMessage(msg.destination.val, reinterpret_cast<const char*>(rep.data()), rep.size());
+      client_receiver->onNewMessage(
+          msg.destination.val, reinterpret_cast<const char*>(rep.data()), rep.size(), msg.destination.val);
```

Comment:
No need to pass the same value twice - let the default value take place.

Reply:
done